### ISSUE TITLE
thx for reply, I tried this, there is a problem, thMake weapon items, like the rifles and launchersto be saved as custom compostitions.

### ISSUE THREAD

**Issue by JinougaF** [zen-mod/ZEN#593]
**Is your enhancement related to a problem?**
The custom composition functions in ZEN would not save the weapon items, like the rifles and launchers, which in Achilles they could be saved as compostitions.

**Solution you'd like:**
A way to make the custom composition functions could save the weapon items so the player can make custom conpositions with them.

**Alternatives you've considered:**
No at the moment

**Additional context:**
No at the momment

**Comment by mharis001** [comment]
By weapon items, do you mean weapon holders because weapons in object inventories are already saved.

**Comment by JinougaF** [comment]
> By weapon items, do you mean weapon holders? Weapons in object inventories are already saved.

Yes I mean weapon holders, the same feature in Achilles could save weapon holders correctly.
